nss-capricious-finches/nutshell
# Api updates

# Description of Proposed Changes
4 different methods are laid out in for the API manager, getall, get single, post something new and update single item. There are comments for what function requires in the arguments.

Steps to Test
Outline the steps to test

```
git fetch --all
git checkout APIupdates
```

Run json server
run grunt in src
open "APImanager" and have a look at the methods
then in main.js test each method. You'll need to create a fake person that is a copy of the user info to post something new.
passing this into the methods should update the API

# List general components of the application that this PR will affect:

API
APIManager.js
main.js

# Mentions @username
@Jonny-Riggs @paulzimmclay 

Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each.
